main.py

```diff
@@ -0,0 +1,167 @@
+import json
+import datetime
+
+isAlarm = True
+mainTmp = -1
+hideoutTmp = -1
+traderTmp = -1
+optionTmp = -1
+skillLevel = 0
+Maker = []
```

Comment:
```suggestion
maker = []
```

이것도 컨벤션이긴 한데 파이썬에서 변수는 소문자로 시작합니다!